eradiate/scenes/spectra/_rgb.py

```diff
@@ -0,0 +1,77 @@
+from typing import MutableMapping, Optional
+
+import attr
+import pint
+import pinttr
+
+import eradiate
+
+from ._core import Spectrum, spectrum_factory
+from ... import unit_context_config as ucc
+from ... import unit_context_kernel as uck
+from ...attrs import documented, parse_docs
+from ...contexts import KernelDictContext, SpectralContext
+from ...exceptions import UnsupportedModeError
+
+
+@spectrum_factory.register(type_id="rgb")
+@parse_docs
+@attr.s
+class RGBSpectrum(Spectrum):
```

Comment:
We should not have the option to introduce a spectrum usable only in one mode: instead, the existing spectrum types should emit a kernel dict compatible with the RGB variant, so that an existing scene can be dropped in a render setup without change.

Reply:
At this point I am not sure if this was introduced by me or if this is an artifact from an old version of Eradiate :thinking: Anyway, I'll remove it.